crates/bevy_render/src/batching/gpu_preprocessing.rs

```diff
@@ -921,3 +941,21 @@ pub fn write_indirect_parameters_buffer(
         .write_buffer(&render_device, &render_queue);
     indirect_parameters_buffer.buffer.clear();
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn instance_buffer_correct_behaviour() {
```

Comment:
```suggestion
    fn instance_buffer_correct_behavior() {
```
Typos-cli apparently hates British English, lol.

Reply:
Yeah hahahaha, Can i check typos-cli localy? And do you know if i have to open a differnt pr if i want to change the implementation from `contains()` to `binary_search()`. Because i think it's better for performance, but i'm unsure and this PR is small anyways.

Reply:
There is a `typos` extension on VSCode from the same project. I'm sure if you look at their repository, they'll have more info.